types/three/examples/jsm/nodes/Nodes.d.ts

```diff
@@ -3,11 +3,12 @@ export * from './core/constants.js';
 
 // core
 export { default as ArrayUniformNode } from './core/ArrayUniformNode.js';
+export { default as AssignNode, assign } from './core/AssignNode.js';
 export { default as AttributeNode, attribute } from './core/AttributeNode.js';
 export { default as BypassNode, bypass } from './core/BypassNode.js';
-export { default as CacheNode, cache } from './core/CacheNode.js';
+export { default as CacheNode, cache, globalCache } from './core/CacheNode.js';
```

Comment:
I don't see `globalCache` being exported here in the source.

Reply:
ah, it's exported from `nodes/core/CacheNode.js` but not exported from `nodes/Nodes`. Thanks.

```suggestion
export { default as CacheNode, cache } from './core/CacheNode.js';
```